config/repositories/repositories.conf

```diff
@@ -14,12 +8,19 @@ prebuild=build_pandoc_bin.sh
 python-genpackagedoc=
 python-extensions-collection=
 python-jsonpreprocessor=
-robotframework-documentation=
+python-pytestlog2db=
+
+robotframework=
```

Comment:
Hi both,
as I remember there was a reason why robotframework was at the very end installed.
One of the packages installed another robotframework. To assure we installed ours at the very end.
Please consider this.
I merge nevertheless that you can check if the build works and proceed stabilization.
Thank you,
 Thomas